canonical-web-and-design/jaas-dashboard
# Persist model grouping via URL

## Done

Persist model grouping via URL

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click different grouping toggles and observe URL changes
- Refresh the page and filter should persist

## Details

Fixes #260
